go/event/event_test.go

```diff
@@ -245,10 +227,8 @@ func TestDispatchUpdate(t *testing.T) {
 
 	ev := &testUpdateEvent{}
 	DispatchUpdate(ev, "hello")
+	assert.True(t, triggered)
```

Comment:
You missed this one, it should have a message passed in.

Reply:
done